.eslintrc.js

```diff
@@ -157,6 +157,24 @@ module.exports = {
                         name: 'jspdf',
                         message: 'Please use pdfmake to generate pdf files.',
                     },
+                    {
+                        name: 'lodash',
+                        message: 'Please use `import [package] from \'lodash/[package]\'` instead',
+                    },
+                ],
+                patterns: [
+                    {
+                        group: ['!lodash/*'],
+                        message: 'Please use `import [package] from \'lodash/[package]\'` instead',
+                    },
+                    {
+                        group: ['*clientSchema*'],
```

Comment:
We have other utils in `clientSchema`, for example `convertEmployeesToOptions` in `BaseTicketForm`. Is this affect it?

Reply:
I'm not sure this utils should be placed in clientSchema. Since `schema` initially told us its the way to receive data on client